tests/e2e/vsan_test.go

```diff
@@ -63,6 +64,10 @@ func (s *VsanTestSuite) SetUpTest(c *C) {
 }
 
 func (s *VsanTestSuite) TearDownTest(c *C) {
+	if len(s.containerList) > 0 {
+		out, err := dockercli.RemoveAllContainers(s.config.DockerHosts[0])
```

Comment:
Running test parallel is an action item for the 0.16 sprint to achieve "Overall time taken by each test run will be ~30minutes." ... we should such `dockercli.RemoveAllContainers` that blocks us moving test run in parallel. 

I don't want to repeat again ... test container is a test item and its test responsibility to remove.

> Running test groups in parallel most likely will simply not work,

Which ever the test is not possible they will be identified and we may put into serial run bucket. 

> Looks like we are already using this in volumeproperty_test.go.

It was a miss from the review and by taking account of above example we should not use this approach. I discussed with @ashahi1  to remove usage of `dockercli.RemoveAllContainers` and remove only the containers created by test. 

/CC @ashahi1